### DISCUSSION TITLE
How to enable nuclei to perform protocol weak passwords and blasting operations, such as ssh, mysql, and other protocols

### DISCUSSION THREAD

**Discussion by ClassBluer** [projectdiscovery/nuclei#3992] [General]
How to enable nuclei to perform protocol weak passwords and blasting operations, such as ssh, mysql, and other protocols

**Comment by Mzack9999** [comment]
@ClassBluer Keep an eye on this PR https://github.com/projectdiscovery/nuclei/pull/3981 it adds a javascript layer for easy high level protocol scripting

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
